src/Extension/AbstractOrderedExtension.php

```diff
@@ -21,7 +22,7 @@
 abstract class AbstractOrderedExtension extends AbstractTypeExtension
 {
     /**
-     * {@inheritdoc}
+     * @param OrderedFormBuilder $builder
```

Comment:
in monorepo, we use FQCN in annotations, but here? I gues it can be like this :slightly_smiling_face: